adcc/OperatorIntegrals.py

```diff
@@ -137,6 +137,26 @@ def nabla(self):
         """
         return self.import_dipole_like_operator("nabla", is_symmetric=False)
 
+    @property
+    def density_dependent_operators(self):
+        if not hasattr(self.provider_ao, "density_dependent_operators"):
+            return {}
```

Comment:
No need. Done exactly the same by the rest of the function.

Reply:
Now obsolete.